src/components/SearchForm/SearchForm.js

```diff
@@ -1,20 +1,19 @@
 import { useState, useContext } from "react";
 import NewsSearchContext from "../../contexts/NewsSearchContext.js";
 import HasSearchedContext from "../../contexts/HasSearchedContext.js";
+import KeywordsContext from "../../contexts/KeywordsContext.js";
 
 function SearchForm() {
   const { handleNewsSearch } = useContext(NewsSearchContext);
   const { setHasSearched } = useContext(HasSearchedContext);
+  const { setKeyword } = useContext(KeywordsContext);
   const [searchInput, setSearchInput] = useState("");
-  const [keywords, setKeywords] = useState([]);
-
-  console.log(keywords);
 
   const handleSubmit = (e) => {
     e.preventDefault();
     handleNewsSearch(searchInput);
     setHasSearched(true);
-    setKeywords(searchInput);
+    setKeyword(searchInput);
     setSearchInput("");
   };
 
```

Comment:
Please, add `value` into the input in line 31


If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components